SourceX/miniwin/misc_msg.cpp

```diff
@@ -530,9 +530,11 @@ bool PeekMessage(LPMSG lpMsg)
 		case SDL_WINDOWEVENT_EXPOSED:
 			lpMsg->message = DVL_WM_PAINT;
 			break;
-		case SDL_WINDOWEVENT_MOVED:
 		case SDL_WINDOWEVENT_RESIZED:
 		case SDL_WINDOWEVENT_SIZE_CHANGED:
+			windowResVer++;
+			break;
+		case SDL_WINDOWEVENT_MOVED:

```

Comment:
Can you elaborate why this extra case was added? At first glance, it seems unrelated to the rest of the changes in the PR.

Reply:
Please see above: https://github.com/diasurgical/devilutionX/pull/955#discussion_r550865112

Reply:
I don't see how that comment explains this TBH. Does moving the window warrant a resize of the cursor? Could you elaborate further?

Please note that my comment was specifically about the new `case SDL_WINDOWEVENT_MOVED` logic.